api/constants.ts

```diff
@@ -0,0 +1 @@
+export const THE_GRAPH_API_KEY = 'db9407a90f0d26076a7adaa0dc3be2f9'
```

Comment:
Let's put this in the `.env` file

Reply:
https://github.com/clober-dex/coupon.finance/pull/1/commits/4845e26ddb33c3f1f6bd735e6e0018ff09660c6f